docker-deploy/proxy/conf.d/default.conf

```diff
@@ -0,0 +1,13 @@
+server {
+	listen 8080;
+	server_name thachban.vaccom.vn;
```

Comment:
This should be passed to the container as a parameter (set in docker-compose.yml?)

Reply:
Thanks, I fixed it